### PR TITLE
sdl2_gfx, add secondary architecture

### PR DESCRIPTION
**Pull request by Begasus**
Fixes also the broken link in developLibDir, added libVersion and defineDebugInfoPackage

### CODE REVIEW

File: media-libs/sdl2_gfx/sdl2_gfx-1.0.4.recipe [hunk 1]
```diff
@@ -71,7 +79,7 @@ TEST()
 {
 	cd Test
 	runConfigure ./configure
-# fails to configure due to missing sdl_gfx
-	make
+# fails to configure due to missing sdl2_gfx
```
Comment (alaviss) [inline media-libs/sdl2_gfx/sdl2_gfx-1.0.4.recipe]: Maybe inspecting the configure log could make this work.

File: media-libs/sdl2_gfx/sdl2_gfx-1.0.4.recipe [hunk 2]
```diff
@@ -6,51 +6,59 @@ which implemented a interpolating rotozoomer for SDL_surfaces."
...
+	sdl2_gfx$secondaryArchSuffix == $portVersion base
 	"
 
 BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
-	devel:libsdl2_2.0
+	devel:libSDL2_2.0$secondaryArchSuffix
 	"
 BUILD_PREREQUIRES="
 	cmd:aclocal
```
[the remark this answers is not in the document]
Comment (Begasus) [reply media-libs/sdl2_gfx/sdl2_gfx-1.0.4.recipe]: Jikes, missed that one? :)

File: media-libs/sdl2_gfx/sdl2_gfx-1.0.4.recipe (hunk 1 above)
Comment (Begasus) [reply media-libs/sdl2_gfx/sdl2_gfx-1.0.4.recipe]: Already checked config.log, no real information there

**Comment by scottmc** [comment]
This good to go now?

**Pull request merged**